.gitignore

```diff
@@ -71,3 +71,7 @@ inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-cpp/release/bin/
 
 # Docker mysql storage
 docker/docker-compose/mysql
+
+inlong-sort/connectors
+plugins
+sort-dist.jar
```

Comment:
It's not wise to add these files, which are used only for your local development env.
All these files exist in `./inlong-distribution/target/` after the build.

Reply:
These files are temporarily created for local debugging in the IDE, So need to be ignored.

Reply:
I have modified all ignore files based on the root directory